geode-assembly/src/test/java/org/apache/geode/BundledJarsJUnitTest.java

```diff
@@ -48,8 +49,10 @@
   public void loadExpectedJars() throws IOException {
     String expectedJarFile =
```

Comment:
Would it be better to move the check into TestUtil.getResourcePath? so that the caller do not have to worry about the platform?